code/lib/types/src/modules/api.ts

```diff
@@ -115,9 +115,20 @@ export interface API_SidebarOptions {
 
 interface OnClearOptions {
   /**
-   *  True when the user dismissed the notification.
+   *  True when the user manually dismissed the notification.
    */
   dismissed: boolean;
+  /**
+   *  True when the notification timed out after the set duration.
+   */
+  timeout: boolean;
```

Comment:
I guess this is to keep backwards compatibility?

A single `reason` property would make more sense

Reply:
Yeah I agree, but I didn't want to touch the existing `dismissed` field.